src/admin/__tests__/fetchOffsets.spec.js

```diff
@@ -40,11 +42,16 @@ describe('Admin', () => {
       )
     })
 
-    test('throws an error if the topic name is not a valid string', async () => {
-      await expect(admin.fetchOffsets({ groupId: 'groupId', topic: null })).rejects.toHaveProperty(
-        'message',
-        'Invalid topic null'
-      )
+    test('throws an error if the topics argument is not a valid list', async () => {
+      await expect(
+        admin.fetchOffsets({ groupId: 'groupId', topics: topicName })
+      ).rejects.toHaveProperty('message', `Expected topic or topics array to be set`)
```

Comment:
We should make sure that this is a non-retriable error, which we can do using the `toThrow` method, which takes a constructor and an optional string that it compares to the message.

```suggestion
      await expect(broker.apiVersions()).rejects.toThrow(
        KafkaJSNonRetriableError,
        `Invalid argument "topics". Expected array of strings, got: "${topicName}"`
      )
```

Reply:
Same for the test below